WIP changes after rebuttal

mathurinm opened the pull request.

mathurinm committed:
add beta* definition

jolars committed:
Specify "coordinate-wise separable"

jolars committed:
Add referance to prox alg

jolars commented:
# Checklist for camera-ready paper

- [x] Clarify introduction (common misconceptions)
- [x] Specify coordinate-wise separability
- [x] Add reference for SLOPE prox
- [x] Add motivation for choice of standardization
- [x] Clarify what density means in tables
- [x] Clarify y axes in figures 6 and 7
- [ ] Mention per-iteration complexity
- [x] Modify abstract regarding "overclaim about improvement"
- [x] Use alternative interval notation in remark 2.1
- [x] Reiterate v = 5
- [x] Add path results to paper (and add acknowledgement to LUNARC for these experiments)

jolars committed:
Add path experiment files

jolars committed:
Specify what density is in the table captions

jolars committed:
Fix linting errors

jolars committed:
Don't rerun if results exist

jolars committed:
Add motivation for standardization

jolars committed:
Clarify "suboptimality" in captions

jolars committed:
Downplay "overclaim" regarding performance in abstract

jolars committed:
Change interval notation

jolars committed:
Reiterate v = 5 in experiments section

jolars committed:
Update AISTATS template

jolars committed:
Add path benchmarks to supplement

jolars commented:
Note that we have an additional page now for the camera-ready version, so we have a bit of space if we want to include something in the main paper. I'm thinking maybe the partial smoothness proposition and/or the path benchmarks, and/or iteration-complexity, and/or extensions to other datafits.

jolars committed:
Add computational spec and acknowledgement to supp

jolars committed:
Remove round-mode=off setting

This clases with changes in siunitx, causing compilation to fail on
newer versions of siunitx. Since 'none' (old 'off') is the default, we
can just unset this option everywhere it's set to 'off'.

mathurinm committed:
intro: number of clusters need not be known, indep of feat order != fused lasso

JonasWallin committed:
complexity disucssion

mathurinm committed:
cosmit computational cost

jolars committed:
Add "for"

jolars committed:
Avoid double "present"

jolars committed:
Change "on" to "in"

jolars committed:
Change "enable" to "enables"

jolars committed:
Change to title case

jolars committed:
Add "step"

jolars committed:
Use cleverref citation

jolars committed:
Remove comma

jolars committed:
Remove space

jolars committed:
Rephrase complexity paragraph

jolars committed:
Add comma to citations

mathurinm committed:
pass grammarly

mathurinm committed:
cosmit proofs
